### ISSUE TITLE
Intellisense mismatch on mixed (lowercase,uppercase) variables

### ISSUE THREAD

**Issue by devel0** [OmniSharp/omnisharp-vscode#3958]
## Environment data
`dotnet --info` output:
```
devel0@tuf:~$ dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.100-preview.6.20318.15
 Commit:    4356580024

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  20.04
 OS Platform: Linux
 RID:         ubuntu.20.04-x64
 Base Path:   /opt/dotnet/sdk/5.0.100-preview.6.20318.15/

Host (useful for support):
  Version: 5.0.0-preview.6.20305.6
  Commit:  4ba9ecaabd

.NET SDKs installed:
  3.1.101 [/opt/dotnet/sdk]
  3.1.202 [/opt/dotnet/sdk]
  3.1.301 [/opt/dotnet/sdk]
  3.1.302 [/opt/dotnet/sdk]
  5.0.100-preview.6.20318.15 [/opt/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.1 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.4 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.5 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.6 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.0-preview.6.20312.15 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.1 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.4 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.5 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.6 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.0-preview.6.20305.6 [/opt/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

VS Code version: 
```
Version: 1.48.0-insider
Commit: d13d2fc56da7a2f8bcad4256212db0661fcbba45
Date: 2020-08-05T05:27:00.961Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.0-42-generic
```

C# Extension version:
```
v.1.22.1
```

## Steps to reproduce

![image](https://user-images.githubusercontent.com/13405008/89775260-bb309a00-db07-11ea-9d2c-4371030383f8.png)

## Expected  behavior

Intellisense should show completion for either `Sb` and `SB` variables

## Actual behavior

Intellisense shows and force completion to `SB`

**Comment by devel0** [comment]
Same problem here, repro if use static System.Math to have access at methods like Abs without prefixing class Math the constant E will taken instead of my local var "e" pressing lowercased e key:

![image](https://user-images.githubusercontent.com/13405008/214336298-da8b261d-e9a3-4880-b54a-7faf9424d3db.png)

**Comment by devel0** [comment]
Solved as of vscode v1.78.2 with omnisharp v1.25.7
